### ISSUE TITLE
[BUG]-Plugin is not able to connect to the org

### ISSUE THREAD

**Issue by rahulAlladaWAL** [forcedotcom/SFDX-Data-Move-Utility#487]
**Describe the bug**
When I try to transfer the data from the source to the target using `sfdx sfdmu:run` (Complete command is - `sfdx sfdmu:run --sourceusername test-1mozktmfkbrt@example.com --targetusername test-7gqm7dljixrv@example.com --quiet --json --path ./transferData/Export_1`), the plugin is not able to connect to the org and is throwing an error: **_Command initialization error: Attempt to connect to test-1mozktmfkbrt@example.com failed. Please, try to refresh your local SFDX CLI connection.._**

**To Reproduce**
Run the command - sfdx sfdmu:run --sourceusername test-1mozktmfkbrt@example.com --targetusername test-7gqm7dljixrv@example.com --quiet --json --path ./transferData/Export_1

**Expected behavior**
Should be able to transfer the data from one org to the other

**export.json**
[export.txt](https://github.com/forcedotcom/SFDX-Data-Move-Utility/files/9250921/export.txt)

**Log file**
**{
  "command": "sfdmu:run",
  "cliCommandString": "sfdx sfdmu:run --sourceusername test-1mozktmfkbrt@example.com --targetusername test-7gqm7dljixrv@example.com --quiet --json --path ./transferData/Export_1",
  "endTime": "2022-08-03T12:26:08.635Z",
  "endTimeUTC": "2022-08-03T06:56:08.635Z",
  "message": "Command initialization error: Attempt to connect to test-1mozktmfkbrt@example.com failed. Please, try to refresh your local SFDX CLI connection..",
  "stack": "Error: Attempt to connect to test-1mozktmfkbrt@example.com failed. Please, try to refresh your local SFDX CLI connection.\n    at ScriptOrg._setupConnection (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\modules\\models\\script_models\\scriptOrg.js:125:27)\n    at ScriptOrg.setupAsync (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\modules\\models\\script_models\\scriptOrg.js:52:20)\n    at Script.setupAsync (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\modules\\models\\script_models\\script.js:180:30)\n    at async RunCommand.setupObjectSetAsync (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\modules\\commands_processors\\runCommand.js:67:9)\n    at async Function.execute (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\modules\\commands_processors\\runCommandExecutor.js:38:17)\n    at async Run.run (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\sfdmu\\lib\\commands\\sfdmu\\run.js:20:9)\n    at async Run._run (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\Documents\\Aduro\\node_modules\\@salesforce\\command\\lib\\sfdxCommand.js:81:40)\n    at async Config.runCommand (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\AppData\\Local\\sfdx\\client\\7.161.0-7a9b98c\\node_modules\\@oclif\\config\\lib\\config.js:173:24)\n    at async SfdxMain.run (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\AppData\\Local\\sfdx\\client\\7.161.0-7a9b98c\\node_modules\\@oclif\\command\\lib\\main.js:27:9)\n    at async SfdxMain._run (C:\\Users\\Lenovo.DESKTOP-PJIBGUQ\\AppData\\Local\\sfdx\\client\\7.161.0-7a9b98c\\node_modules\\@oclif\\command\\lib\\command.js:43:20)",
  "startTime": "2022-08-03T12:26:03.121Z",
  "startTimeUTC": "2022-08-03T06:56:03.121Z",
  "status": 2,
  "statusString": "COMMAND_INITIALIZATION_ERROR",
  "timeElapsedString": "00h 00m 05s 514ms "
}**

**Comment by hknokh** [comment]
Hello

Checked this. For me it's everytime connecting well.

- Do you really have an org created using the example.org domain name ?
- Can you try to refresh the local SFDX org connection using sfdx auth:web:login ?
- Also check the connection by performing the below command:
sfdx force:org:display -u test-1mozktmfkbrt@example.com --json
and send me your result here.

**Comment by rahulAlladaWAL** [comment]
This is the JSON I have got.
```
{
  "status": 0,
  "result": {
    "id": "00D1k0000007sP3EAI",
    "devHubId": "masterportal@advisors.com",
    "accessToken": "00D1k0000007sP3!AQwAQOV55uRsASABMc7mpacrOInd4RSV.OiiNFKCHECnxfpjO1i5AFhR0icvZssbCJ5IzqsbJxxFQguibIBYTxLCXAunsw2W",
    "instanceUrl": "https://inspiration-enterprise-3204-dev-ed.my.salesforce.com",
    "username": "test-1mozktmfkbrt@example.com",
    "clientId": "PlatformCLI",
    "status": "Active",
    "expirationDate": "2022-08-17",
    "createdBy": "masterportal@advisors.com",
    "edition": "Developer",
    "namespace": "ad",
    "orgName": "july18",
    "createdDate": "2022-07-18T04:20:46.000+0000",
    "alias": "https://inspiration-enterprise-3204-dev-ed.my.salesforce.com"
  },
  "warnings": [
    "This command will expose sensitive information that allows for subsequent activity using your current authenticated session.\nSharing this information is equivalent to logging someone in under the current credential, resulting in unintended access and escalation of privilege.\nFor additional information, please 
review the authorization section of the https://developer.salesforce.com/docs/atlas.en-us.234.0.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm"
  ]
}
```

**Comment by hknokh** [comment]
Ok. understand. I will fix it ASAP

**Comment by hknokh** [comment]
I've just uploaded the fix for this.
Not sure for the fix since it's working for me well
Update and try again

**Comment by rahulAlladaWAL** [comment]
Worked like a gem. Thanks!

**Comment by tokyo0709** [comment]
@hknokh We are still experiencing this issue on 4.15.1 We have an instance where we can connect into other sandboxes just fine but when we try to go to Prod we get the same error above. Notably, I've tried in our same org under my DMU setup and it works into Prod but I am running 4.14.4

**Comment by hknokh** [comment]
Can't reproduce this.
Checked with your sfdx force:org:display JSON it's recognized as connected.
Please, debug the source on your side and let me know if found something.

**Comment by tokyo0709** [comment]
@hknokh Any idea how I would go about debugging the source on my side? I'm not sure I know where to start with that.

**Comment by hknokh** [comment]
@tokyo0709 
This is the article:
https://help.sfdmu.com/plugin-basics/basic-usage/debugging

**Comment by hknokh** [comment]
@tokyo0709 hello
Do you still experiencing the issue ? Did you debug it ?

**Comment by hknokh** [comment]
Hope all is good. Closing for now.

**Comment by tokyo0709** [comment]
@hknokh Just wanted to give a heads up, we didn't get a chance to debug it, but after several attempts of reconfiguration found that we had to uninstall the SFDX CLI completely along with the plugin and reinstall and after we did that we were able to connect to Production just fine.

We'd tried several other options before this like, updating the CLI, updating the plugin, logging out of Production, reauthorizing production, updating VS Code, uninstalling/reinstalling the Salesforce Extension pack, and none of them did the trick. Wonder if there was some saved configuration somewhere that got corrupted.
